install/kubernetes/helm/istio-init/templates/job-crd-all.yaml

```diff
@@ -12,7 +12,11 @@ spec:
       serviceAccountName: istio-init-service-account
       containers:
       - name: istio-init-crd-all
+{{- if (hasKey .Values.global "kubectl") and (hasKey .Values.global.kubectl "image") }}
```

Comment:
this logic is a little repetitious and difficult to understand (imo).  why not just create a new flag for the kubectl image name?

Reply:
+1, this logic is difficult to understand.